### PR TITLE
Cache google sheets data in localStorage

### PR DESCRIPTION
**Pull request by yuvipanda**
Raw data from Google Sheets is cached in localStorage,
and the application is seeded from this data the next time
the application is loaded. This makes subsequent loads much
faster, especially if the data hasn't changed at all.

Ref #9 #11

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,24 +1,48 @@
...
     },
     mounted(){
-        d3.csv('https://docs.google.com/spreadsheets/d/e/2PACX-1vTTUD7zK1YRnkzoaVoD33VGMUywCaFhS9gRZXa2_SAEb5PnIne7YWxWqy8k1vYvdc5ocT75_KleDmSX/pub?gid=0&single=true&output=csv').then(function (data) {
-            dataList = data.map(d => {
-                d['phone'] = d['phone'].split('/').map(n => n.trim());
-                d['type'] = d['type'].split('/').map(n => n.trim());
-                return d;
-            });
-            list.contactList = [...dataList];
+        d3.csv(sheetsUrl).then(function (data) {
```
Comment (prtksxna) [inline js/main.js:43]: Just to be clear, this still loads the data from the Google Sheet even if it finds it in `localStorage` and then updates the `localStorage` and the UI when it does, right?

If so could you add a comment to this effect here.

**Review by prtksxna — COMMENTED** [review]
Thanks for this @yuvipanda!

File: js/main.js (hunk 1 above)
Comment (yuvipanda) [reply js/main.js:43]: @prtksxna done!

**Comment by prtksxna** [comment]
This looks good to me :shipit: 

@siddharthashok @wazidshah do you want to take a look at this too?

**Pull request merged**